cmd/bb_storage/main.go

```diff
@@ -40,6 +52,32 @@ func main() {
 		log.Fatal(http.ListenAndServe(*webListenAddress, nil))
 	}()
 
+	pe, err := prometheus_exporter.NewExporter(prometheus_exporter.Options{
+		Namespace: "bb_storage",
+		Registry: prometheus.DefaultRegisterer.(*prometheus.Registry),
+	})
+	if err != nil {
+		log.Fatalf("Failed to create the Prometheus stats exporter: %v", err)
```

Comment:
Nitpicking  but the rest of the code uses log.Fatal exclusively, maybe sticking with it for consistency sake would be better?